packages/react-vanilla-components/src/components/CheckBox.tsx

```diff
@@ -37,7 +37,7 @@ const CheckBox = (props: PROPS) => {
   };
 
   return (
-    <div className="cmp-adaptiveform-checkbox" data-cmp-is="adaptiveFormCheckBox" data-cmp-visible={visible} data-cmp-enabled={enabled}>
+    <div className={appliedCssClassNames ? `cmp-adaptiveform-checkbox ${appliedCssClassNames}` : 'cmp-adaptiveform-checkbox'} data-cmp-is="adaptiveFormCheckBox" data-cmp-visible={visible} data-cmp-enabled={enabled}>
```

Comment:
Can you simplify this by using  className={`cmp-adaptiveform-checkbox ${appliedCssClassNames || ''}`}

Reply:
Done!